source/features/pr-earliest-tag-after-merge.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import oneTime from 'onetime';
+import TagIcon from 'octicon/tag.svg';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import fetchDom from '../libs/fetch-dom';
+import features from '../libs/features';
+import {getRepoURL} from '../libs/utils';
+
+const getEarliestTag = cache.function(async (mergeCommit: string): Promise<string[] | undefined> => {
```

Comment:
```suggestion
const getEarliestTag = cache.function(async (mergeCommit: string): Promise<[string, string] | undefined> => {
```

Or maybe this works, I never remember:

```suggestion
const getEarliestTag = cache.function(async (mergeCommit: string): Promise<[string, string]?> => {
```

Also it can just be called `commit`